i18n/es.yaml

```
@@ -0,0 +1,259 @@
+
+- id: translations-available
+  translation: "Traducciones"
+
+- id: read-in-language
+  translation: "Leer esta entrada en %s"
+
+- id: also-available-in
+  translation: "También disponible en"
+
+- id: current-language
+  translation: "Idioma actual"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)